src/app/products/product-list.spec.ts

```diff
@@ -161,7 +161,7 @@ describe("Product Details",() => {
 
 
       it('should to products if id > 1', fakeAsync(() => {
-        const id = PRODUCTS[1].productId;
+        const id = PRODUCTS[1].id;
         const path = '/products/' + 0;
```

Comment:
we generally don't use these routes directly. We create global variables for routing.